src/Datadog.Trace.ClrProfiler.Managed/Integrations/Testing/XUnitIntegration.cs

```diff
@@ -31,14 +31,13 @@ public static class XUnitIntegration
 
         private static readonly IntegrationInfo IntegrationId = IntegrationRegistry.GetIntegrationInfo(nameof(IntegrationIds.XUnit));
         private static readonly Vendors.Serilog.ILogger Log = DatadogLogging.GetLogger(typeof(XUnitIntegration));
-        private static readonly FrameworkDescription _runtimeDescription;
+        private static readonly FrameworkDescription RuntimeDescription;
```

Comment:
`XUnitIntegration` doesn't need to cache this anymore.

Reply:
Done